WORKSPACE

```diff
@@ -48,10 +48,11 @@ git_repository(
 
 http_archive(
     name = "io_bazel_rules_go",
-    url = "https://github.com/bazelbuild/rules_go/releases/download/0.7.1/rules_go-0.7.1.tar.gz",
-    sha256 = "341d5eacef704415386974bc82a1783a8b7ffbff2ab6ba02375e1ca20d9b031c",
+    url = "https://github.com/bazelbuild/rules_go/releases/download/0.8.1/rules_go-0.8.1.tar.gz",
+    sha256 = "90bb270d0a92ed5c83558b2797346917c46547f6f7103e648941ecdb6b9d0e72",
 )
 
+
```

Comment:
@alexeagle please delete this extra line return before merging. I have fixed it in the bot already, but it won't "undo" this mistake automatically here.

Reply:
haha it's a bot PR so I get very lazy. If there's no formatter for this file, then there's no enforcement.

thanks!